app/browser/tools/activityHub.js

```diff
@@ -56,15 +56,31 @@ class ActivityHub {
 	 * @param {number} state 
 	 */
 	setMachineState(state) {
-		instance.whenReady().then((inst) => {
-			if (state === 1) {
-				this.refreshAppState(inst.controller, state);
-			} else {
-				inst.controller.appStateService.setMachineState(state);
+		const teams2IdleTracker = instance.getTeams2IdleTracker();
```

Comment:
this is going to apply no matter if we got the optInV2 as it triggers on events. We will need a config option to be pass in here.

Reply:
ignore, this is from 1.4.16 and I think is worth the risk as I don't think MS is going to handle react and angular working at the same time